.gitignore

```diff
@@ -13,3 +13,4 @@ default*
 .vscode/
 /.dashboard_asset_cache
 /.tools/bin
+docs/swagger*
```

Comment:
Create a new file `placeholder.go` in the directory and specify other files to ignore it.